tests/braille-specs/Makefile.am

```
@@ -12,6 +12,7 @@ EXTRA_DIST =					\
 	bn.yaml					\
 	chr-us-g1_harness.yaml			\
 	cop-eg.yaml				\
+	critical-apparatus.yaml		\
```

Did you mean `hbo.yaml` here?

---

I'll fix this

---

Oh wait, this is no mistake. This is commit https://github.com/liblouis/liblouis/commit/4d9ce05da9e242f4c81f84f9c833dacf8d51e158

---

Yes, but in the Makefile it should mention `he-common.yaml` then and not `critical-apparatus.yaml`

```suggestion
	he-common.yaml		\
```

---

Sorry, I meant this is commit cbb5d5c48e687758f8576eef6e83786e7698b107.